README.md

```diff
@@ -23,140 +23,19 @@ With Finch, you can leverage these existing projects without chasing down all th
-
-```sh
-brew install --cask finch
-```
-
-Once the installation is complete, `finch vm init` is required once to set up the underlying system. This initial setup usually takes about a minute.
-
-```sh
-finch vm init
-INFO[0000] Initializing and starting Finch virtual machine...
-..
-INFO[0067] Finch virtual machine started successfully
-```
-
-### Running containers and building images
-
-You can now run a test container. If you're familiar with container development, you can use the `run` command as you'd expect.
-
-```sh
-finch run --rm public.ecr.aws/finch/hello-finch
-```
-
-If you're new to containers, that is so exciting! Give the command above a try after you've installed and initialized Finch. The `run` command pulls an image locally if it's not already present, and then creates and runs a container for you. Note the handy `--rm` option will delete the container instance once it's done executing.
-
-To build an image, try a quick example from the finch client repository.
-
-```sh
-git clone https://github.com/runfinch/finch.git
-cd finch/contrib/hello-finch
-finch build . -t hello-finch
-..
-```
-
-Again if you're new to containers, you just built a container image. Nice!
-
-The `build` command will work with the build system (the Moby Project's BuildKit in Finch's case) to create an OCI image from a Dockerfile, which is a special sort of recipe for creating an image. This image can then be used to create containers. You can see your locally pulled and built images with the `finch images` command.
-
-Finch makes it easy to build and run containers across architectures with the `--platform` option. When used with the `run` command, it will create a container using the specified architecture. For example, on an Apple Silicon M1 system, `--platform=amd64` will create a container and run processes within it using an x86-64 architecture.
-
-```sh
-uname -ms
-Darwin arm64
-
-finch run --rm --platform=amd64 public.ecr.aws/amazonlinux/amazonlinux uname -ms
-Linux x86_64
-```
-
-You can also use the `--platform` option with builds, making it easy to create multiplatform images.
-
-## Working with Finch
-
-We have plans to create some more documentation and tutorials here geared toward users who are new to containers, as well as some tips and tricks for more advanced users. For now, if you're ready to kick the tires, please do! You'll find most commands and options you're familiar with from other tools to present, and as you'd expect (or, as they are [documented upstream with nerdctl](https://github.com/containerd/nerdctl#command-reference)). Most of the commands we use every day are covered, including volume and network management as well as Compose support. If Finch doesn't do something you want it to, please consider opening an Issue or a Pull Request.
+- [Installing Finch](https://runfinch.com/docs/managing-finch/macos/installation/)
+- [Building your first container
+  image](https://runfinch.com/docs/getting-started/installation/)
```

Comment:
```suggestion
  image](https://runfinch.com/docs/getting-started/building-images/)
```

Reply:
Good spot! Included in the latest commit.